test/controllers/site_controller_test.rb

```diff
@@ -147,7 +147,7 @@ def test_permalink
 
     get permalink_path(:code => "wBz3--", :changeset => 4)
     assert_response :redirect
-    assert_redirected_to :controller => :browse, :action => :changeset, :id => 4, :anchor => "map=3/4.8779296875/3.955078125"
+    assert_redirected_to :controller => :changesets, :action => :show, :id => 4, :anchor => "map=3/4.8779296875/3.955078125"
```

Comment:
This can use `changeset_path` now I think?

Reply:
Updated. This change could have been done before too. Same for some other `assert_redirected_to`.